### ISSUE TITLE
Update the Spring Boot version

### ISSUE THREAD

**Issue by Buzzardo** [spring-guides/gs-accessing-data-rest#40]
An ongoing task is to keep the Spring Boot version current in the build files:

To do so, update the following files:

complete/build.gradle
complete/pom.xml
initial/build.gradle
initial/pom.xml

**Comment by GeorgePanaretos** [comment]
Issued is resolved by:
https://github.com/spring-guides/gs-accessing-data-rest/commit/1669a729a9088c4f4c5017cbea58f3e1f6725d35

**Comment by Buzzardo** [comment]
Yes, it is. We leave one of these open all the time, though. As soon as the next patch release comes out, someone can fix this "issue" again. GitHub doesn't have an always-open issue mode.
